src/components/Disclosure/README.md

```diff
@@ -11,7 +11,7 @@ Disclosure component that shows and hides enclosed content.
 | disabled        | `Boolean`                     |          | `false` | Disabled state                                           |
 | defaultExpanded | `Boolean`                     |          | `false` | Default opening state                                    |
 | expanded        | `Boolean`                     |          |         | Controlled opening state                                 |
-| arrowPosition   | `left` `right`                |          | `left`  | Control position                                         |
+| arrowPosition   | `left` `right` `start` `end`  |          | `start` | Control position                                         |
```

Comment:
We suppose left and right are being deprecated, so they don’t need to be documented